src/openapi/streaming/parser/wrappers/protobuf-wrappers.ts

```diff
@@ -0,0 +1,33 @@
+import type { IWrapper, Message } from 'protobufjs';
+
+export default {
+    register: (wrappers: Record<string, IWrapper>) => {
+        if (!wrappers['.google.protobuf.Timestamp']) {
+            wrappers['.google.protobuf.Timestamp'] = {
+                fromObject(object) {
+                    return this.fromObject(object);
+                },
+
+                toObject(message: Message): Record<string, any> {
+                    // casting type since it's not able to tel TS that the message contains proper types
+                    const { seconds, nanos } = (message as unknown) as {
+                        nanos?: number;
+                        seconds?: number;
+                    };
+
+                    // Date with support for nano precision
+                    const date = new Date(
+                        Number(seconds) * 1000 +
+                            Math.floor(Number(nanos) / 1000000),
+                    );
+
+                    // @ts-expect-error FIXME TS is complaining that a string is returned,

```

Comment:
Looks like something we intended to fix before v9 release

Reply:
Discussed it with @michax and this is expected. I updated the comment.